components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -25,6 +26,10 @@
 SEARCH_WORKER_COMPONENT_NAME = "search_worker"
 WEBUI_COMPONENT_NAME = "webui"
 
+# target names
+ALL_TARGET = ""
```

Comment:
This is minor but feels like naming this as `ALL_TARGET_NAME` to be consistent with other target names would pay off at some point.